src/parser/pegen.ts

```diff
@@ -1336,6 +1337,16 @@ export function singleton_seq(p: Parser, a: AST): AST[] {
 //     return seq;
 // }
 
+export function seq_insert_in_front(p: Parser, a: any, seq: any[] | null) {
+    console.assert(a !== null);
```

Comment:
There's also written an `assert()` function at the top of this file that'll throw an `AssertionError`